test/fixtures/JSX/empty-expression-container.js

```diff
@@ -0,0 +1 @@
+<body>{}</body>
```

Comment:
This test demonstrates an inconsistency in Flow (which is not followed in this pull request).

Consider the following two cases, `<a>{ }</a>` and `<a>{b}</a>`, and what Flow produces. In the first case, the location range for the inner JSXEmptyExpression node includes the curly braces, i.e. the same as the location range of the enclosing JSXExpressionContainer node. In the second case, the Identifier node for `b` obviously does not include the curly braces.